tsfile/src/test/java/org/apache/iotdb/tsfile/file/header/PageHeaderTest.java

```diff
@@ -119,8 +122,8 @@ private PageHeader deSerialized(int offset) {
     FileInputStream fis = null;
     PageHeader header = null;
     try {
-      fis = new FileInputStream(new File(PATH));
-      header = PageHeader.deserializeFrom(DATA_TYPE, fis.getChannel(), offset, true);
+      TsFileInput input = new DefaultTsFileInput(Paths.get(PATH));
```

Comment:
Hm... But DefaultTsFileInput class actually use a FileChannel... 
So, will you override it in tsfile-spark?

Reply:
Yes. In TsFile-Spark-Connector I use HDFSInput to override TsFileInput. In HDFSInput FSDataInputStream and FileStatus are used instead of FileChannel to implement the interfaces in TsFileInput. That's why I need to change FileChannel in these functions to TsFileInput.

Reply:
+1. But I suggest that u record this requirement on JIRA

Reply:
JIRA IoTDB-63: Use TsFileInput instead of FileChannel as the input parameter of some functions